### PR TITLE
Add thermal-ir tutorial notebook placeholders

### PR DESCRIPTION
**Pull request by spestana**

### CODE REVIEW

**Comment by scottyhq** [comment]
@spestana don't forget to `Restart Kernel and Clear all Outputs` before committing (https://github.com/snowex-hackweek/website/pull/43/checks?check_run_id=2766769812), JupyterBook will execute the notebook and save outputs for the website. This keeps the repository small and versioning clean since we don't have large images and cell counts etc in the notebooks.

File: book/tutorials/thermal-ir/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Thermal Infrared Remote Sensing of Snow
+
+**At the conclusion of this tutorial, you will be able to:**
+- Understand the differences between sources of thermal IR observations, the advantages and disadvantages of each, and types of research questions we can address with those observations
+- Access and visualize airborne and satellite thermal infrared imagery, then scale imagery of different spatial resolutions for comparison
+- Visualize point and raster thermal infrared datasets together, then compute error statistics between point and raster datasets
```
Comment (scottyhq) [inline book/tutorials/thermal-ir/index.md:6]: This is great. One nice feature of the JupyterBook rendering is using call-out boxes or "admonitions", which you might want to try out https://jupyterbook.org/content/content-blocks.html#notes-warnings-and-other-admonitions 

```{admonition} Learning objectives

- Access and visualize airborne and satellite thermal infrared imagery, then scale imagery of different spatial resolutions for comparison
- Visualize point and raster thermal infrared datasets together, then compute error statistics between point and raster datasets
- etc
```

File: book/_toc.yml [hunk 2]
```diff
@@ -34,6 +34,10 @@
           sections:
             - file: tutorials/sar/sentinel1
             - file: tutorials/sar/uavsar
+        - file: tutorials/thermal-ir/index
+          sections:
+            - file: tutorials/thermal-ir/thermal-ir-1
```
Comment (scottyhq) [inline book/_toc.yml:39]: Looks like you need to add `thermal-ir-0.ipynb` here (which downloads the necessary file for subsequent notebooks. How large are these files? We could also put them here as a backup https://github.com/snowex-hackweek/tutorial-data

**Comment by scottyhq** [comment]
@spestana looks like you just have to update the _toc.yml file with your new notebook names. Would you be OK to merge this today? Can follow up next week with additional changes if you want

File: book/tutorials/thermal-ir/index.md (hunk 1 above)
Comment (spestana) [reply book/tutorials/thermal-ir/index.md]: Thanks, I've added a few of these admonition blocks!

File: book/_toc.yml (hunk 2 above)
Comment (spestana) [reply book/_toc.yml]: Updated to use a single notebook for the main tutorial with datasets downloaded from s3. I have extra notebooks and .py files that I didn't include in the toc with the idea that that those are accessible through other hyperlinks or when the repo is cloned to jupyterhub.

**Comment by spestana** [comment]
> 
> 
> @spestana looks like you just have to update the _toc.yml file with your new notebook names. Would you be OK to merge this today? Can follow up next week with additional changes if you want

This should be ok to merge now. The only changes I would need to make now would be if something was broken or if I have significant typos :)

**Comment by scottyhq** [comment]
This looks fantastic @spestana ! Thanks so much for all the effort. I fixed some figure markup. There are still some warnings that we can ignore. basically markup to files that are not *rendered* on the website do not get linked, for example: 
```
* [Downloading datasets through the NASA EarthData API](./thermal-ir-data-download.ipynb)
  * [earthdata_api.py](./earthdata_api.py)
```

Results in build logs like the following b/c files are not copied to the website. The links do work as you'd expect running the notebooks on the hub though
```
writing output... [100%] tutorials/thermal-ir/thermal-ir-tutorial              
132
/home/jovyan/book/tutorials/thermal-ir/index.md:16: WARNING: 'myst' reference target not found: ./thermal-ir-data-download.ipynb
133
/home/jovyan/book/tutorials/thermal-ir/index.md:17: WARNING: 'myst' reference target not found: ./earthdata_api.py
```

**Review by scottyhq — APPROVED** [review]

**Pull request merged**